src/Syntax.hs

```diff
@@ -26,15 +26,26 @@ newtype Typed = Typed Typed deriving Data
 
 data family Var a
 
+class InModule v where
+  inModule :: v -> v -> v
```

Comment:
Can you not use `Monoid` for this - or, well, `Semigroup`?

Reply:
I'm 90% sure I can replace this with an empty space. I don't think I ended up using it at all.

Reply:
Oh.